packages/permissionless/accounts/etherspot/utils/encodeCallData.ts

```diff
@@ -0,0 +1,89 @@
+import {
+    type Address,
+    type Hex,
+    concatHex,
+    encodeAbiParameters,
+    encodeFunctionData,
+    toHex
+} from "viem"
+import { EtherspotExecuteAbi } from "../abi/EtherspotExecuteAbi"
+import { CALL_TYPE, EXEC_TYPE } from "../constants"
+import { getExecMode } from "./getExecMode"
+
+export const encodeCallData = (
+    _tx:
```

Comment:
Can you change `_tx -> tx`. We use `_` to denote unused variables.

Reply:
Changes done.